stata_kernel/kernel.py

```diff
@@ -53,9 +57,15 @@ def __init__(self, *args, **kwargs):
 
         config = ConfigParser()
         config.read(os.path.expanduser('~/.stata_kernel.conf'))
+
         self.execution_mode = config['stata_kernel']['execution_mode']
         self.stata_path = config['stata_kernel']['stata_path']
 
+        try:
+            self.graphdir = config['stata_kernel']['graphdir']
+        except:
+            self.graphdir = tempfile.mkdtemp()
+
```

Comment:
I updated this in dev: 
https://github.com/kylebarron/stata_kernel/blob/0450471d04f2a30a5d4ff9e10659ee811e409a15/stata_kernel/kernel.py#L37-L38

`StataSession` creates it if it doesn't exist:
https://github.com/kylebarron/stata_kernel/blob/0450471d04f2a30a5d4ff9e10659ee811e409a15/stata_kernel/stata_session.py#L36-L37